src/init/azext_init/_help.py

```diff
@@ -10,4 +10,11 @@
 helps['init'] = """
     type: command
     short-summary: It's an effortless setting up tool for configs.
```

Comment:
short-summary: Set Azure CLI user configurations by responding to questions with multiple choice answers

long-summary: |
The Azure CLI offers `az init` to help you set global configurations through an interactive question-answer format.  Global settings are grouped into bundles that help you optimize your settings for scenarios such as "interaction" or "automation".  To familiarize yourself with common configurations and select available options, answer the step-by-step questions, and `az init` will do the rest!

The purpose of `az init` is not to replace the `az config` command, but to simplify the setting of configurations by offering an interactive question-answer experience.  While both `az inti` and `az config` modify the same configuration file, `az init` is designed to only set the most common configurations.  For more granular control of all available configurations, use `az config`.  For more information on configuring your Azure CLI environment, see https://docs.microsoft.com/en-us/cli/azure/azure-cli-configuration.